test/blackbox/CMakeLists.txt

```diff
@@ -194,9 +194,9 @@ macro(add_blackbox_gtest)
     endif()
 endmacro()
 
-option(RTPS_API_TESTS "Enable tests using RTPS API" ON)
-option(FASTRTPS_API_TESTS "Enable tests using FastRTPS API" ON)
-option(FASTDDS_PIM_API_TESTS "Enable tests using FastDDS API" OFF)
+option(RTPS_API_TESTS "Enable tests using RTPS API" OFF)
```

Comment:
RTPS is part of the API and has not been deprecated, so we should not have this OFF by default, should we?

Reply:
We should definitely NOT have this OFF by default

Reply:
Re-enabled these tests on commit 4cc6eb3